### PR TITLE
Automation test: Switching network regression

### PR DESCRIPTION
**Pull request by MilanVojnovic95**
# Summary

This is a regression test for switching network, and this scenario has many times caused that application crash (black screen).
All details of the scenario and steps can be found at this [link](https://docs.google.com/spreadsheets/d/129Vr4cxi9z1McOAXHy2gU3RD4xL_IwwU7V9_T0YYxLU/edit#gid=204937697&range=62:62).

  # To Test

- [ ] Run test for SwitchingNetworkTests.ts with yarn synpress:run {path to file}
- [ ] The test should be performed successfully

### CODE REVIEW

File: tests/synpress/specs/transactionfull/swap/SwapTests.ts [hunk 1]
```diff
@@ -214,15 +206,11 @@ describe('Swapping tests', () => {
 
     cy.confirmMetamaskTransaction({})
 
-    SwapPage.getTransactionConfirmedModal()
```
Comment (niemam29) [inline tests/synpress/specs/transactionfull/swap/SwapTests.ts:217]: same

**Review by denciu — COMMENTED** [review]
Looks good, have nothing more to add than what @niemam29 mentioned. Resolve these and I will be happy to approve.

**Comment by MilanVojnovic95** [comment]
@niemam29 @denciu 
Fixed required things, please take a look.

File: tests/synpress/specs/transactionfull/swap/SwapTests.ts [hunk 2]
```diff
@@ -90,7 +90,10 @@ describe('Swapping tests', () => {
     SwapPage.getTransactionConfirmedModal()
       .should('be.visible')
       .should('contain.text', 'Transaction Submitted')
+<<<<<<< HEAD
+=======
 
+>>>>>>> automation-tests-ecobridge
```
Comment (denciu) [inline tests/synpress/specs/transactionfull/swap/SwapTests.ts:96]: Leftovers from resolving conflicts

**Review by denciu — CHANGES_REQUESTED** [review]

**Review by niemam29 — CHANGES_REQUESTED** [review]

File: tests/synpress/specs/transactionfull/swap/SwapTests.ts [hunk 3]
```diff
@@ -205,6 +205,7 @@ describe('Swapping tests', () => {
     SwapPage.swap().confirmSwap()
 
     cy.confirmMetamaskTransaction({})
+    SwapPage.getTransactionConfirmedModal().should('be.visible').should('contain.text', 'Transaction Submitted')
```
[the remark this answers is not in the document]
Comment (MilanVojnovic95) [reply tests/synpress/specs/transactionfull/swap/SwapTests.ts:208]: will be solved 👍

**Pull request closed without merging**